### DISCUSSION TITLE
Platform Electric Book Works

### DISCUSSION THREAD

**Discussion by mrchristian** [singlesourcepub/community#10] [Q&A]
Anyone used the Markdown Boo platform 'Electric Book Works' https://electricbookworks.com/

**Comment by xldrkp** [comment]
Nice hint, thanks @mrchristian! I had a look at their business model. Looks like one has to do it like that to earn money from such a service. I'd prefer to go on open source first and see what develops from that. But as they have a technical single source publishing approach, too, we can learn from that.

**Comment by mrchristian** [comment]
I assumed they were 100% open-source, I see the use Prince for PDFs which is not FOSS - thats understandable if you need its features. This is there repo https://github.com/electricbookworks/electric-book - I'll ask around.

**Reply by xldrkp** [reply]
Ah, thanks for pointing that out, hadn't found something like "Fork me on GitHub" or the like.

**Reply by mrchristian** [reply]
I also prefer it when people are very clear and up front about FOSS approach. Its a little burried
